aws/rust-runtime/aws-types/src/region.rs

```diff
@@ -107,14 +107,36 @@ impl From<&'static str> for SigningRegionSet {
 
 impl SigningRegionSet {
     /// Create a `SigningRegionSet` from a static str.
+    ///
+    /// ## Example
+    ///
+    /// ```rust
+    /// use aws_types::region::SigningRegionSet;
+    ///
+    /// // When setting multiple regions, use a comma-separated list.
+    /// let region_set = SigningRegionSet::from_static("us-east-1, us-east-2, us-west-1");
```

Comment:
are there supposed to be spaces in this list?

Reply:
Sources say: "no".

I removed this function anyhow.